docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -28,310 +28,174 @@ The first step to installing on a new machine is to install :term:`NCEPLIBS` (ht
 
    * CMake v3.15+ is needed for building NCEPLIBS, but versions as old as 3.12 can be used to build NCEPLIBS-external, which contains a newer CMake that can be used for the rest of the build.
 
-For both Linux and macOS, you will need to set the stack size to "unlimited" (if allowed) or the largest possible value.
+For both Linux and macOS systems, users will need to set the stack size to "unlimited" (if allowed) or the largest possible value.
 
 .. code-block:: console
 
    # Linux, if allowed
    ulimit -s unlimited
 
-   # macOS, this corresponds to 65MB
+   # MacOS, this corresponds to 65MB
    ulimit -S -s unlimited
 
-For Linux systems, as long as the above software is available, you can move on to the next step: installing the :term:`NCEPLIBS-external` package.
+For Linux systems, as long as the above software is available, users can move on to the next step: installing the :term:`HPC-Stack` package according to the instructions in :numref:`Chapter %s <InstallBuildHPCstack>`.
 
-For macOS systems, some extra software is needed: ``wget``, ``coreutils``, ``pkg-config``, and ``gnu-sed``.
-It is recommended that you install this software using the Homebrew package manager for macOS (https://brew.sh/):
-
-* brew install wget
+For MacOS systems, some extra software is needed: ``gcc@11``, ``cmake``, ``make``, ``wget``, ``coreutils``, and ``pkg-config``. It is recommended that users install this software using the `Homebrew <https://brew.sh/>`__ package manager for MacOS:
 
+* brew install gcc@11
 * brew install cmake
-
+* brew install make
+* brew install wget
 * brew install coreutils
-
 * brew install pkg-config
 
-* brew install gnu-sed
+.. 
+   COMMENT: Is this still accurate? It seems like we should delete the last 2 and add openssl@3, Lmod, curl, libtiff.
 
```

Comment:
@mkavulich, willmayfield, can you comment on @gspetro-NOAA's question here?

Reply:
@natalie-perlin do you know?